lib/Swagger/src/SiteAttribute/Api/SiteAttributeApi.php

```diff
@@ -0,0 +1,535 @@
+        }
+        // parse inputs
+        $resourcePath = "/site/{siteId}/attr/{attrName}";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('*_/_*'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // path params
+        if ($site_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "siteId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($site_id),
+                $resourcePath
+            );
+        }
+        // path params
+        if ($attr_name !== null) {
+            $resourcePath = str_replace(
+                "{" . "attrName" . "}",
+                $this->apiClient->getSerializer()->toPathValue($attr_name),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
+        } elseif (count($formParams) > 0) {
+            $httpBody = $formParams; // for HTTP post (form)
+        }
+        // this endpoint requires API key authentication
+        $apiKey = $this->apiClient->getApiKeyWithPrefix('X-Wikia-AccessToken');
+        if (strlen($apiKey) !== 0) {
+            $headerParams['X-Wikia-AccessToken'] = $apiKey;
+        }
+        // this endpoint requires API key authentication
+        $apiKey = $this->apiClient->getApiKeyWithPrefix('X-Wikia-UserId');
+        if (strlen($apiKey) !== 0) {
+            $headerParams['X-Wikia-UserId'] = $apiKey;
+        }
+        // make the API Call
+        try {
+            list($response, $statusCode, $httpHeader) = $this->apiClient->callApi(
+                $resourcePath,
+                'DELETE',
+                $queryParams,
+                $httpBody,
+                $headerParams,
+                null,
+                '/site/{siteId}/attr/{attrName}'
+            );
+
+            return array(null, $statusCode, $httpHeader);
+        } catch (ApiException $e) {
+            switch ($e->getCode()) {
+            }
+
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation getAllAttributes
+     *
+     * Returns all available attributes for the specified siteId
+     *
+     * @param int $site_id The ID of the site (required)
+     * @return \Swagger\Client\SiteAttribute\Models\AllSiteAttributesHalResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getAllAttributes($site_id)
+    {
+        list($response) = $this->getAllAttributesWithHttpInfo($site_id);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)